src/internal/connector/exchange/service_iterators_test.go

```diff
@@ -1144,11 +1144,12 @@ func (suite *ServiceIteratorsSuite) TestFilterContainersAndFillCollections_incre
 	}
 
 	table := []struct {
-		name     string
-		getter   mockGetter
-		resolver graph.ContainerResolver
-		dps      DeltaPaths
-		expect   map[string]endState
+		name            string
+		getter          mockGetter
+		resolver        graph.ContainerResolver
+		dps             DeltaPaths
+		expect          map[string]endState
+		skipForNonDelta bool
```

Comment:
nit: the name here is slightly confusing
```suggestion
		skipWhenForcedNoDeltas bool
```